lib/methods/import.js

```diff
@@ -600,7 +638,7 @@ module.exports = self => {
     }) {
       // TODO: Find a more efficient way to compare only relatedIds
       // (we need to find a way to keep autopublish published IDs in that case)
-      if (duplicatedIds && docIds) {
+      if ((duplicatedIds && duplicatedIds.size) && docIds) {
```

Comment:
```suggestion
      if (duplicatedIds?.size) && docIds) {
```

optional

Reply:
Stupid me, I used default value to make it less verbose while I could do that. thx

Reply:
Oh no I'm stuck with my way of doing things I think I'll show you.